tests/servers/tests.py

```diff
@@ -98,6 +100,22 @@ def test_live_server_url_is_class_property(self):
 
 
 class LiveServerViews(LiveServerBase):
+    def test_protocol(self):
+        """
+        Ensure launched server serves with HTTP 1.1.
+        """
+        with captured_stdout() as debug_output:
+            conn = HTTPConnection(LiveServerViews.server_thread.host, LiveServerViews.server_thread.port)
+            try:
+                conn.set_debuglevel(1)
+                conn.request('GET', '/example_view/', headers={"Connection": "keep-alive"})
+                conn.getresponse()
+                conn.request('GET', '/example_view/', headers={"Connection": "close"})
+                conn.getresponse()
+            finally:
+                conn.close()
+        self.assertIn("HTTP/1.1 200 OK", debug_output.getvalue())
```

Comment:
Including the reply prefix, e.g. "reply: 'HTTP/1.0 200 OK" might be slightly more clear?
Otherwise looks good. Worth a sentence in the release notes?